code/__HELPERS/unsorted.dm

```diff
@@ -1106,7 +1106,9 @@ B --><-- A
 	A.cut_overlay(O)
 
 /proc/get_random_station_turf()
-	return safepick(get_area_turfs(pick(GLOB.the_station_areas)))
+	var/list/turfs = get_area_turfs(pick(GLOB.the_station_areas))
+	if (length(turfs))
+		return pick(get_area_turfs(pick(GLOB.the_station_areas)))
```

Comment:
just pick(turfs) here ?

Reply:
yeah pick() is fine with empty lists you just get null

Reply:
pick() from an empty list is a runtime